pkg/controllermanager/controller/seed/seed.go

```diff
@@ -111,19 +139,39 @@ func NewSeedController(
 		AddFunc: seedController.seedLifecycleAdd,
 	})
 
+	seedInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: seedController.seedAdd,
```

Comment:
Probably we should also react on update, right? I'm imaging a world in which the default domain secrets can have "seed selectors", i.e., they would only apply for seeds having specific labels, so we should reconcile this when the labels change. OTOH, if it's too much effort for now, we can also postpone it to the time when we introduce such seed selectors.

Reply:
> I'm imaging a world in which the default domain secrets can have "seed selectors", i.e., they would only apply for seeds having specific labels

Wouldn't it be the other way around i.e., the seed have a secret selector / secret refs for default domains? I think there won't be seed selectors for secrets, as this is not supported by the API, right? 👀

Reply:
> Wouldn't it be the other way around i.e., the seed have a secret selector / secret refs for default domains? 

Hm, the problem with this would be that a `Seed` can "fish" secrets from other seeds (that it's not supposed to see) by simply changing this selector.

> I think there won't be seed selectors for secrets, as this is not supported by the API, right? 👀

Yeah, we would probably have to continue the path that we use today to make the secret carrying additional information about the DNS (e.g., domain, provider, etc.): Labels/Annotations. Such "seed selector" would probably be an annotation on the secret?

Reply:
To me it sounds like we should rather make `SecretBindings` fit for such use-cases instead of working with selector-annotations. Anyways, like you suggest, let's discuss this topic when we take care about it. We can still adjust the enqueuing logic later.